crud.py

```diff
@@ -0,0 +1,54 @@
+import schemas
+from sqlalchemy.orm import Session
+from database import models
+
+
+def get_author(db: Session, author_id: int) -> models.Author:
+    return db.query(models.Author).filter(models.Author.id == author_id).first()
+
+
+def get_authors(db: Session, skip: int = 0, limit: int = 100) -> list[models.Author]:
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def create_author(db: Session, author: schemas.AuthorCreate) -> models.Author:
+    db_author = models.Author(name=author.name, bio=author.bio)
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
+
+
+def get_author_by_name(db: Session, name: str) -> models.Author:
+    return db.query(models.Author).filter(models.Author.name == name).first()
+
+
+def get_book(db: Session, book_id: int) -> models.Book:
+    return db.query(models.Book).filter(models.Book.id == book_id).first()
+
+
+def get_books(db: Session, skip: int = 0, limit: int = 100) -> list[models.Book]:
+    return db.query(models.Book).offset(skip).limit(limit).all()
```

Comment:
Improvement (fix only if you have a desire)
Create a BookService class to enforce some architecture in your project
Move crud methods inside this class
`db` should be instance attribute, and this way you won't need to pass it to every crud method